ide/provisioning/install-debian-packages.sh

```diff
@@ -42,11 +43,12 @@ apt-get install $APTOPTS make g++ \
   librsvg2-bin xsltproc \
   imagemagick gettext \
   mesa-common-dev libgl1-mesa-dev libegl1-mesa-dev \
-  fonts-dejavu
+  fonts-dejavu \
+  libnetcdf-dev
```

Comment:
What for?

Reply:
The upcoming Skysight integration will add this requirement, but as long as it's not merged we don't need it. Removed.